src/applications/facility-locator/actions/index.js

```diff
@@ -220,6 +220,7 @@ export const genBBoxFromAddress = (query) => {
             payload: {
               ...query,
               context: zipCode,
+              inProgress: true,
```

Comment:
This took a bit of digging but the sequence of actions was flipping this `inProgress` state on the third step which would cause the loading indicator to toggle off before the whole sequence related to search was done.

I found this is reusing action types across several action creators which makes it hard to track this kind of bug down. Might put a ticket in to fix this. 

![screen shot 2019-02-28 at 19 09 10](https://user-images.githubusercontent.com/4998130/53611623-7ec7e300-3b8c-11e9-8e35-bf4db548bf2d.png)

Reply:
This stops some of the problem but I think there's another action doing a similar thing and it's causing the loading indicator to flash out and in. Think I'll just make a ticket for cleaning up these action sequences a bit instead of digging into it as part of this issue. 

![facility-locator-search-focus](https://user-images.githubusercontent.com/4998130/53651608-3a782980-3c04-11e9-8907-6a45d5c243e6.gif)

Reply:
Issue created: https://github.com/department-of-veterans-affairs/vets.gov-team/issues/17200